### PR TITLE
fix(3rdparty): Add a CI job to check 3rdparty integrity 

### PR DESCRIPTION
**Pull request by nickvergessen**
* Should prevent bugs like https://github.com/nextcloud/server/pull/44377#issuecomment-2039651912

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by nickvergessen** [comment]
/backport 8aae7fbc9caaa7378bf9650304b25acf2c45b3f1 to stable29

**Comment by nickvergessen** [comment]
/backport 8aae7fbc9caaa7378bf9650304b25acf2c45b3f1 to stable28

**Comment by nickvergessen** [comment]
/backport 8aae7fbc9caaa7378bf9650304b25acf2c45b3f1 to stable27

**Comment by nickvergessen** [comment]
/backport 8aae7fbc9caaa7378bf9650304b25acf2c45b3f1 to stable26

File: .github/workflows/block-outdated-3rdparty.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# This workflow is provided via the organization template repository
```
Comment (susnux) [inline .github/workflows/block-outdated-3rdparty.yml:1]: Is it?

**Comment by nickvergessen** [comment]
/backport 8aae7fbc9caaa7378bf9650304b25acf2c45b3f1 to stable25

**Comment by nickvergessen** [comment]
/backport 8aae7fbc9caaa7378bf9650304b25acf2c45b3f1 to stable24

**Review by susnux — APPROVED** [review]
Looks good and makes sense! :rocket:

**Review by bigcat88 — APPROVED** [review]

**Pull request merged**